common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -661,18 +683,47 @@
                 this.showLanguageMenu = true;
 
                 $.each(languages, function(code, label) {
-                    var li = $('<li data-lang-code="' + code + '" />'),
-                        link = $('<button class="control control-lang">' + label + '</button>');
+                    li = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML(
+                            '<li data-lang-code="{code}" />'
+                        ),
+                        {
+                            code: code
+                        }
+                    ).toString();
+                    
+                    link = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML(
+                            '<button class="control control-lang" aria-pressed="false">{label}</button>'
+                        ),
+                        {
+                            label: label
+                        }
+                    ).toString();
+                    
+                    liObj = $.parseHTML(li);
```

Comment:
I needed a jQuery object to interact with. I tried several different methods (not the above, though) but this was the only thing that worked without re-writing the entire script. I will try Andy's suggestion tomorrow and see where that gets me. I'm also still trying to diagnose the RequireJS dependency for the HtmlUtils.